src/components/molecules/Settings/Workspace/Plugin/PluginAccordion/index.test.tsx

```diff
@@ -5,18 +5,18 @@ import PluginAccordion from "./index";
 describe("Accordion should display header and body", () => {
   const items = [
     {
-      id: "hogehoge",
-      thumbnail: `${process.env.PUBLIC_URL}/sample.svg`,
+      thumbnailUrl: `${process.env.PUBLIC_URL}/sample.svg`,
       title: "Sample",
       isInstalled: true,
       bodyMarkdown: "# Hoge ## Hoge",
+      author: "reearth",
     },
     {
-      id: "fugafuga",
-      thumbnail: `${process.env.PUBLIC_URL}/sample.svg`,
+      thumbnailUrl: `${process.env.PUBLIC_URL}/sample.svg`,
       title: "Sample2",
       isInstalled: false,
       bodyMarkdown: "# Fuga ## Fuga",
+      author: "reearth",
     },
   ];
   test("Accordion should be rendered", () => {
```

Comment:
Please just out of curiosity:
1- What type of tests is this? `import { render, screen } from "@reearth/test/utils";` is only used twice in all Re:Earth project.
2- If it is using `getByText`, I guess it's checking the UI (E2E) so why not using integrated Cypress library to test?

Reply:
1. That is a component test for react component. We use `react-testing-library` and can test if the components are rendered as expected.  Our project uses some special like loading i18n from YAML files, receive style from theme providers, etc. So we need to mock that kind of thing and that's why I defined the custom `render` function. Also, it can receive the mock of GQL data and can be used for components that have dependencies on GQL.

Reply:
2. Cypress is used for E2E test. This is the component test like a unit test for components.